keygen/src/keygen.rs

```diff
@@ -161,16 +170,16 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                 check_for_overwrite(&outfile, &matches);
             }
 
-            let phrase = rpassword::prompt_password_stdout("Mnemonic recovery phrase: ").unwrap();
+            let phrase = rpassword::prompt_password_stderr("Mnemonic recovery phrase: ").unwrap();
             let mnemonic = Mnemonic::from_phrase(phrase.trim(), Language::English)?;
             let seed = Seed::new(&mnemonic, NO_PASSPHRASE);
             let keypair = keypair_from_seed(seed.as_bytes())?;
 
-            let serialized_keypair = write_keypair(&keypair, outfile)?;
+            let serialized_keypair = write_keypair_file(&keypair, outfile)?;
             if outfile == "-" {
                 println!("{}", serialized_keypair);
             } else {
-                println!("Wrote recovered keypair to {}", outfile);
+                eprintln!("Wrote recovered keypair to {}", outfile);
             }
```

Comment:
This section needs to be rewritten to match your changes in "new"

Reply:
Nice catch!  I must've lost that in my, "Oh just let me make history pretty..." rebase

Reply:
864c3b8 oughta do it